org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/TypePattern.java

```diff
@@ -187,7 +187,7 @@ final ASTNode internalGetSetChildProperty(ChildPropertyDescriptor property, bool
 	 * </ul>
 	 * @exception UnsupportedOperationException if this operation is used other than JLS19
 	 * @exception UnsupportedOperationException if this expression is used with previewEnabled flag as false
-	 * @Deprecated(forRemoval = true, since="2024-07")
+	 * @deprecated(forRemoval = true, since="2025-03")
```

Comment:
See https://github.com/eclipse-platform/eclipse.platform/blob/master/docs/Eclipse_API_Central_Deprecation_Policy.md#java-api

It should be **annotation**, not javadoc comment

Reply:
Okay, now got the point. I have pushed the changes. @iloveeclipse could you please review this PR if time permits.